src/roman_datamodels/datamodels/_datamodels.py

```diff
@@ -131,30 +129,7 @@ def to_parquet(self, filepath):
         pq.write_table(table, filepath, compression=None)
 
 
-class _DataModel(DataModel):
-    """
-    Exists only to populate the __all__ for this file automatically
-        This is something which is easily missed, but is important for the automatic
-        documentation generation to work properly.
-    """
-
-    __slots__ = ()
-
-    def __init_subclass__(cls, **kwargs):
-        """Register each subclass in the __all__ for this module"""
-        super().__init_subclass__(**kwargs)
-
-        # Don't register private classes
-        if cls.__name__.startswith("_"):
-            return
-
-        if cls.__name__ in __all__:
-            raise ValueError(f"Duplicate model type {cls.__name__}")
-
-        __all__.append(cls.__name__)
-
-
-class _RomanDataModel(_DataModel):
```

Comment:
I noticed that all this along with the __all__ was blocking my IDE from back tracing to the datamodel objects here. This isn't necessary because the  _-prefix will block against those being part of __all__.